kyo-core/shared/src/main/scala/kyo/Atomic.scala

```diff
@@ -1,8 +1,15 @@
 package kyo
 
-import java.util.concurrent.atomic as j
```

Comment:
I don't know why Metals started complaining about this import after I added the scaladocs. I've used the full name instead.